modules/auxiliary/gather/ldap_query.rb

```diff
@@ -174,9 +175,9 @@ def generate_rex_tables(entries, format)
 
       case format
       when 'table'
-        print_status(tbl.to_s)
+        print_line(tbl.to_s)
       when 'csv'
-        print_status(tbl.to_csv)
+        print_line(tbl.to_csv)
```

Comment:
Changed these to `print_line` so that the `[*] ` prefix doesn't throw the table's title alignment off with its underline.